source/hardware/controllers/ion_discover_dj_pro.rst

```diff
@@ -0,0 +1,99 @@
+ION Discover DJ Pro
+===================
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro
+
+Ion Discover DJ Pro is a 2-deck controller with integrated audio interface and iPhone dock.
+
+.. figure:: ../../_static/controllers/ion_discover_dj_pro.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: ION Discover DJ Pro (schematic view)
+   :figclass: pretty-figures
+
+   ION Discover DJ Pro (schematic view)
+
+-  `Mixxx User Forum <https://mixxx.discourse.group/t/ion-discover-dj-pro-mapping-script/12120>`__
+
+.. versionadded:: 2.2.5
+
+Compatibility
+-------------
+
+This controller is USB MIDI class compliant, so it is plug-and-play on all platforms without having to install
+any special drivers. The manufacturer’s page only mentions Windows and Mac but it’s confirmed working on Linux
+as well.
+
+Controller Mapping
+------------------
+
+.. note:: Only controls are documented that are known to be working.
+
```

Comment:
This should be removed, this is redundant with the "Known Issues" section.

```suggestion
```

Reply:
Done